test/distributed/org/apache/cassandra/distributed/test/InternodeEncryptionEnforcementTest.java

```diff
@@ -134,19 +125,14 @@ public void testOutboundConnectionsAreRejectedWhenAuthFails() throws IOException
     public void testOutboundConnectionsAreInterruptedWhenAuthFails() throws IOException, TimeoutException
     {
         Cluster.Builder builder = createCluster(AllowFirstAndRejectOtherOutboundAuthenticator.class);
+        final ExecutorService executorService = Executors.newSingleThreadExecutor();
+
         try (Cluster cluster = builder.start())
```

Comment:
nit: it is safer to close the `ExecutorService` along with `cluster`. Otherwise, if the body of `try` throws, only `cluster` is closed, but `executorService.shutdown()` will not be called. 

You can have multiple `closable`s in the try-with-resources statement, and remove `executorService.shutdown()` at the end of the test.
```
try (Cluster cluster = builder.start(); AutoCloseable es = executorService::shutdown)
```